frameworks/Java/officefloor/officefloor-async.dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM maven:slim as maven
+FROM maven:latest as maven
 WORKDIR /officefloor
 COPY src src
-WORKDIR /officefloor/src/woof_benchmark_async
+WORKDIR /officefloor/src/woof_benchmark
 RUN mvn -B clean package
 
-FROM openjdk:slim
+FROM openjdk:latest
 WORKDIR /officefloor
 COPY --from=maven /officefloor/src/woof_benchmark_async/target/woof_benchmark_async-1.0.0.jar server.jar
```

Comment:
This is the only one failing now because I think you meant line 4 to be `WORKDIR /officefloor/src/woof_benchmark_async`

```
officefloor-async: Exception: COPY failed: stat officefloor/src/woof_benchmark_async/target/woof_benchmark_async-1.0.0.jar: file does not exist
```

Reply:
Also, could you tie down the version of `openjdk`. We don't want to use `:latest` as that can introduce breaking changes after successfully passing tests here and it helps keep results reproducible.

Reply:
Yep, no worries, will do now :+1: